packages/python-packages/api-stub-generator/apistub/_apiview.py

```diff
@@ -29,17 +29,18 @@ class ApiView:
     :param str: ver_string
     """
 
-    def __init__(self, nodeindex, pkg_name="", pkg_version=0, ver_string="", namespace = ""):
+    def __init__(self, nodeindex, pkg_name="", pkg_version="", namespace = ""):
         self.Name = pkg_name
-        self.Version = pkg_version
-        self.VersionString = ver_string
+        self.Version = 0
+        self.VersionString = ""
```

Comment:
Was this intentional? Wouldn't we want this meta-data?

Reply:
Yup. APIView tool sets these two fields when it parses ast.